composer/packages/distribution/webpack.config.js

```diff
@@ -21,50 +14,31 @@ module.exports = {
         libraryTarget: 'umd'
     },
     resolve: {
-        alias: {
-            '../../theme.config$': path.join(
-                __dirname, 'node_modules/@ballerina/theme/src/themes/default/theme.config')
-        },
         extensions: ['.tsx', '.ts', '.js', '.json']
     },
     module: {
         rules: [{
                 test: /\.css$/,
                 use: ['style-loader', 'css-loader']
             },
-            {
-                use: ExtractDefaultThemeCSS.extract({
-                    use: [
-                        {
-                            loader: 'css-loader', options: {
-                            sourceMap: true
-                            }
-                        },
-                        {
-                            loader: 'less-loader', options: {
-                            sourceMap: true
-                            }
-                        }
-                    ]
-                }),
-                test: /(themes).default.*\.less$/,
-            },
             {
                 exclude: /(themes).*\.less/,
                 test: /\.less$/,
                 use: [
-                        'style-loader', 
-                        {
-                            loader: 'css-loader', options: {
+                    'style-loader',
+                    {
+                        loader: 'css-loader',
+                        options: {
                             sourceMap: true
-                            }
-                        },
-                        {
-                            loader: 'less-loader', options: {
+                        }
+                    },
+                    {
+                        loader: 'less-loader',
```

Comment:
do we still need less loader here?

Reply:
Ermm.. not really. Kept it for safe side. But since we are enforcing the styling through theming, we can take out the entire style loader I guess.

Reply:
+1 for removal as it will prevent less imports within diagram.

Reply:
Removed.